dapr/proto/runtime/v1/dapr.proto

```diff
@@ -37,6 +37,9 @@ service Dapr {
 
   // Gets secrets from secret stores.
   rpc GetSecret(GetSecretRequest) returns (GetSecretResponse) {}
+
+  // Perform transactions for a specified store
+  rpc PerformTransaction(MultiStateRequest) returns (google.protobuf.Empty) {}

```

Comment:
I would move this under DeleteState rpc definition (L31)
I am not sure `PerformTransaction` is a good naming. 
The purpose of this api is to execute multiple state requests.

When it comes to request protobuf message naming, please rename MultiStateRequest to `[RPC Method]Request` convention. e.g. `ExecuteStateReuqests

@orizohar @mchmarny @msfussell @yaron2 @amanbha do you have the better idea for this naming?

Reply:
ping @orizohar @mchmarny @msfussell @yaron2 @amanbha Do you have the better idea for `PerformTransaction` naming?

Reply:
+1, `[RPC Method]Request` follows the used convention

Reply:
Personally I'd like to see `ExecuteStateTransaction(StateTransactionRequest)`.

Reply:
I spoke to @mchmarny offline, one suggested name for this api is `ExecuteStateTransaction` following the convention `<action><object><subject>`. 

I used to refer to [googleapis repo](https://github.com/googleapis/googleapis/tree/master/google). They don't use `Perform` verb in their rpc apis. So I suggest to use `Execute` action verb instead of `Perform`.

Reply:
@yaron2 I didn't recognize you replied :) I like the rpc method name.

But can we use `ExecuteStateTransactionRequest` instead of `StateTransactionRequest` for request protobuf message to be consistent with the other apis?

Reply:
Yeah that's fine

Reply:
@chinzhiweiblank Please rename `PerformTransaction(MultiStateRequest)` to
`ExecuteSateTransaction(ExecuteSateTransactionRequest)` and move it to under DeleteState api.

Reply:
Renamed.